demos/projects/ESPRESSIF/esp32/README.md

```diff
@@ -187,7 +184,7 @@ The output should show traces similar to:
 <summary>See more...</summary>
 
 ```shell
-$ idf.py -p /dev/ttyUSB0 monitor
+$ idf.py -B "C:\espbuild" -p /dev/ttyUSB0 monitor
```

Comment:
This should be kept as is I think. The long path issue is only or mostly on Windows.
And if we verify it is an applicable workaround, the path should be something like /espbuild since it's Linux.

Reply:
let's keep as it is.